### PR TITLE
Save all resumes

### PR DESCRIPTION
**Pull request by asavienko**

### CODE REVIEW

File: api/routes/parseAllResume.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+const saveAllResume = require("../services/saveAllResume/saveAllResume");
+
+const parseAllResume = async (req, res) => {
+  try {
+    const result = await saveAllResume();
+    res.json({ data: result });
```
Comment (bohdanSnappy) [inline api/routes/parseAllResume.js]: return smth?

File: api/services/getAuthToken/initBrowser.js [hunk 2]
```diff
@@ -1,11 +1,11 @@
 const puppeteer = require("puppeteer");
-const browserHeadless = require("../constants/userData").browserHeadless;
+const browserHeadless = require("../../toDellete/constants/constants").browserHeadless;
 
 const initBrowser = async () => {
   const browser = await puppeteer.launch({
     headless: browserHeadless
   });
-  return await browser.newPage();
+  return await browser;
```
Comment (bohdanSnappy) [inline api/services/getAuthToken/initBrowser.js]: remove await

File: api/services/saveAllResume/saveAllResume.js [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+  const responseDB = await collectionReports.insertOne({
+    startDate: new Date(),
+    addedResume: 0,
+    emptyPages: 0,
+    resumesId: [],
+    status: "started"
+  });
+  const reportId = responseDB.insertedId;
+  try {
```
Comment (bohdanSnappy) [inline api/services/saveAllResume/saveAllResume.js]: check try catch, try to use operator "if" in mongodb

Comment (bohdanSnappy) [reply api/services/saveAllResume/saveAllResume.js]: https://docs.mongodb.com/manual/reference/operator/aggregation/cond/

File: api/services/saveAllResume/saveReport.js [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+) => {
+  collectionReports.updateOne(
+    { _id: reportId },
+    {
+      $inc: { addedResume: insertedCount, emptyPages },
+      $push: { insertedIds: { $each: insertedIds } },
+      $set: { status: "executes" }
+    }
+  );
+  return emptyPages ? { status: "finishing" } : { status: "executes" };
```
Comment (bohdanSnappy) [inline api/services/saveAllResume/saveReport.js]: move emptyPages condition to top

File: api/routes/parseAllResume.js (hunk 1 above)
Comment (asavienko) [reply api/routes/parseAllResume.js]: I don't know what function has to return

File: server.js [hunk 5]
```diff
@@ -25,46 +33,13 @@ app.get("/cvlist", async (req, res) => {
...
-  //todo up to date data from api
-  try {
-    const collection = await connectCollection("rabotaua", "dictionaryCity");
-    const result = await collection.find().toArray();
-    res.json({ confirmation: "success", data: result });
-  } catch (e) {
-    res.json({
+
+    //todo replace error response
+    /*res.json({
```
Comment (bohdanSnappy) [inline server.js]: remove comment

File: src/services/cvRequests.js [hunk 6]
```diff
@@ -12,7 +12,7 @@ const getTotalCv = async ({ regionId: regionid, keywords }) => {
     moveability: 0,
     sort: "date"
   });
-
+  console.log(url);
```
Comment (bohdanSnappy) [inline src/services/cvRequests.js]: remove console.log

**Pull request merged**